### PR TITLE
hardening/372_date_objects

### PR DESCRIPTION
**Pull request by frbattid**
* Implements the issue https://github.com/telefonicaid/fiware-cygnus/issues/372
* 100% unit tests passed
```
Results :
Tests run: 49, Failures: 0, Errors: 0, Skipped: 0
```
* (unofficial) e2e tests passed
```
{ "_id" : ObjectId("55489a680dc73311f6deadb0"), "recvTime" : ISODate("2015-05-05T10:24:40Z"), "entityId" : "Room1", "entityType" : "Room", "attrName" : "temperature", "attrType" : "centigrade", "attrValue" : "26.5" }
```
* Assignee @fgalan but LGTM from @gtorodelvalle is also required. Comments from @iariasleon are welcome as well.
* **DO NOT MERGE UNTIL PR https://github.com/telefonicaid/fiware-cygnus/pull/377 IS MERGED INTO DEVELOP**.

### CODE REVIEW

File: doc/design/OrionMongoSink.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Assuming `mongo_username=myuser` as configuration parameter, then `OrionMongoSin
     > use vehicles
     switched to db vehicles
     > show collections
-    4wheels
+    4wheels
```
Comment (frbattid) [inline doc/design/OrionMongoSink.md]: Github magic...

File: doc/design/OrionMongoSink.md [hunk 2]
```diff
@@ -112,4 +112,4 @@ Fermín Galán Márquez (fermin.galanmarquez@telefonica.com) **[Contributor and
 <br>
 Germán Toro del Valle (german.torodelvalle@telefonica.com) **[Contributor]**
 <br>
-Iván Arias León (ivan.ariasleon@telefonica.com) **[Quality Assurance]**
\ No newline at end of file
+Iván Arias León (ivan.ariasleon@telefonica.com) **[Quality Assurance]**
```
Comment (frbattid) [inline doc/design/OrionMongoSink.md]: Github magic...

File: doc/design/OrionMongoSink.md (hunk 1 above)
Comment (fgalan) [inline doc/design/OrionMongoSink.md]: NTC

File: doc/design/OrionMongoSink.md (hunk 2 above)
Comment (fgalan) [inline doc/design/OrionMongoSink.md]: NTC

**Comment by fgalan** [comment]
LGTM (passing the ball to @gtorodelvalle for final LGTM + merge + delete branch)

**Comment by gtorodelvalle** [comment]
Yo have my blessing and my LGTM :) Merging :) If you have further comments, @iariasleon , please do not hesitate to open a new issue... ;) Thanks!

**Pull request merged**